docs/dev.rst

```diff
@@ -88,6 +88,195 @@ To run the webapp service, do:
 The Symbols Service webapp is at: http://localhost:3000
 
 
+How to
+======
+
+What services are running in a local dev environment
+----------------------------------------------------
+
+============  ====  =============================================
+service       port  description
+============  ====  =============================================
+frontend      3000  Javascript proxy for webapp--use with browser
+web           8000  Django webapp--use with APIs
+localstack    4566  S3 emulation service
+db            5432  Postgres database
+redis         6379  Redis service
+fakesentry    8090  Sentry emulation service
+oidcprovider  8080  SSO emulation service
+statsd        8081  Grafana / statsd
+============  ====  =============================================
```

Comment:
Having this table is realllly helpful. I wish I had created it ages ago. I can never remember which fake service is on which port.